Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Get_entity_alerts_by_MITRE_tactics.yml

```diff
@@ -0,0 +1,1971 @@
+        simple: Hunt Privilege Escalation techniques - Cortex XDR - Endpoint Investigation
+      tags:
+        simple: Defense_Evasion
+      customfields: {}
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "58_26_yes": 0.89
+    },
+    "paper": {
+      "dimensions": {
+        "height": 6455,
+        "width": 1020,
+        "x": -190,
+        "y": -1070
+      }
+    }
+  }
+inputs:
+- key: HuntReconnaissanceTechniques
+  value:
+    simple: "True"
+  required: false
+  description: Set to True to hunt for identified alerts with MITRE Reconnaissance techniques.
+  playbookInputQuery:
+- key: HuntInitialAccessTechniques
+  value:
+    simple: "True"
+  required: false
+  description: Set to True to hunt for identified alerts with MITRE Access techniques.
+  playbookInputQuery:
+- key: HuntExecutionTechniques
+  value:
+    simple: "True"
+  required: false
+  description: Set to True to hunt for identified alerts with MITRE Execution techniques.
+  playbookInputQuery:
+- key: HuntPersistenceTechniques
+  value:
+    simple: "True"
+  required: false
+  description: Set to True to hunt for identified alerts with MITRE Persistence techniques.
+  playbookInputQuery:
+- key: HuntPrivilegeEscalationTechniques
+  value:
+    simple: "True"
+  required: false
+  description: Set to True to hunt for identified alerts with MITRE Privilege Escalation techniques.
+  playbookInputQuery:
+- key: HuntDefenseEvasionTechniques
+  value:
+    simple: "True"
+  required: false
+  description: Set to True to hunt for identified alerts with MITRE Defense Evasion techniques.
+  playbookInputQuery:
+- key: HuntDiscoveryTechniques
+  value:
+    simple: "True"
+  required: false
+  description: Set to True to hunt for identified alerts with MITRE Discovery techniques.
+  playbookInputQuery:
+- key: HuntLateralMovementTechniques
+  value:
+    simple: "True"
+  required: false
+  description: Set to True to hunt for identified alerts with MITRE Lateral Movement techniques.
+  playbookInputQuery:
+- key: HuntCollectionTechniques
+  value:
+    simple: "True"
+  required: false
+  description: Set to True to hunt for MITRE Collection techniques identified alerts.
+  playbookInputQuery:
+- key: HuntCnCTechniques
+  value:
+    simple: "True"
+  required: false
+  description: Set to True to hunt for identified alerts with MITRE Command and Control techniques.
+  playbookInputQuery:
+- key: HuntImpactTechniques
+  value:
+    simple: "True"
+  required: false
+  description: Set to True to hunt for identified alerts with MITRE Impact techniques.
+  playbookInputQuery:
+- key: HuntCredentialAccessTechniques
+  value: {}
+  required: false
+  description: Set to True to hunt for identified alerts with MITRE Credential Access techniques.
+  playbookInputQuery:
+- key: timeRange
+  value:
+    simple: 2 hours
```

Comment:
shouldn't it be "2 hours **ago**"?

Reply:
so it said "ago" on the description but the "time frame" input in "xdr-get-alerts" asks without "ago" so should i will just fix the description